github/issues.go

```diff
@@ -44,6 +44,7 @@ type Issue struct {
 	Milestone        *Milestone        `json:"milestone,omitempty"`
 	PullRequestLinks *PullRequestLinks `json:"pull_request,omitempty"`
 	Repository       *Repository       `json:"repository,omitempty"`
+	RepositoryURL    *string           `json:"repository_url,omitempty"`
```

Comment:
Please move it under `HTMLURL` so that it’s grouped together with all `url` fields.